test/verify/check-connection

```diff
@@ -737,6 +737,7 @@ until pgrep -f cockpit-bridge.*--privileged; do sleep 1; done
         self.allow_journal_messages("Refusing to render service to dead parents.")
         self.allow_journal_messages(".*No authentication agent found.*")
         self.allow_journal_messages(".*Peer failed to perform TLS handshake.*")
+        self.allow_journal_messages("cannot reauthorize identity\\(s\\): unix-user:admin unix-user:fedora")
```

Comment:
FYI: this is easier written with raw strings: `r"foo\(..."`. (No need to push again, of course)